src/script/src/python/pyo3/builtins.rs

```diff
@@ -203,11 +203,6 @@ macro_rules! simple_vector_fn {
     };
 }
 
-#[pyfunction]
```

Comment:
Why delete this function?

Reply:
> Why delete this function?

Used `add_class::<PyVector>()` instead, so this `vector` function overlap with `vector`'s constructor, so remove it